tests/zeek/udp.zeek

```diff
@@ -17,7 +17,7 @@ public type Message = unit {
 # @TEST-START-FILE udp-test.evt
 protocol analyzer spicy::UDP_TEST over UDP:
     parse with UDPTest::Message,
-    port 31337/udp;
+    port 31337/udp-31340/udp;
```

Comment:
That syntax with the two protocols feels a little bit odd, we could instead do `31337-31340/udp`. But I see from the code that parsing the current syntax is easier, and it's a rare use case to begin with; so fine leaving as is, just wanted to note.

Reply:
Actually, just reading the doc PR: should we move the range parsing into the plural case: `ports 31337/udp-31340/udp`?

Reply:
The way I had refactored this already supported specifying either a port or a range, in both `port` or `ports` instructions. I have clarified the docs in zeek/spicy#852.